examples/xgboost_sklearn/train_sklearn.py

```diff
@@ -0,0 +1,48 @@
+from pprint import pprint
+
```

Comment:
Awesome example! Can we add a brief README to this directory explaining what this example covers? E.g. Usage of XGBoost's scikit-learn integration with MLflow Tracking, particularly autologging?